src/main/java/uk/gov/hmcts/sptribs/common/notification/CaseWithdrawnNotification.java

```diff
@@ -92,23 +98,23 @@ public void sendToApplicant(final CaseData caseData, final String caseNumber) {
     }
 
     private NotificationResponse sendEmailNotification(final String destinationAddress, final Map<String, Object> templateVars) {
-        NotificationRequest request = notificationHelper.buildEmailNotificationRequest(
+        final NotificationRequest request = notificationHelper.buildEmailNotificationRequest(
             destinationAddress,
             templateVars,
             TemplateName.CASE_WITHDRAWN_EMAIL);
         return notificationService.sendEmail(request);
     }
 
     private void sendLetterNotification(Map<String, Object> templateVarsLetter) {
-        NotificationRequest letterRequest = notificationHelper.buildLetterNotificationRequest(
+        final NotificationRequest letterRequest = notificationHelper.buildLetterNotificationRequest(
             templateVarsLetter,
             TemplateName.CASE_WITHDRAWN_POST);
         notificationService.sendLetter(letterRequest);
     }
 
     private void addCaseClosedTemplateVars(CaseData caseData, Map<String, Object> templateVars) {
-        CloseCase closeCase = caseData.getCloseCase();
-        String additionalDetail = StringUtils.isNotEmpty(closeCase.getAdditionalDetail())
+        final CloseCase closeCase = caseData.getCloseCase();
+        final String additionalDetail = StringUtils.isNotEmpty(closeCase.getAdditionalDetail())
```

Comment:
Please add a unit test for this where closeCase.AdditionalDetail is not provided